beta/azure-container-apps/ReadMe.md

```diff
@@ -0,0 +1,279 @@
+# [Beta] Deploy 1Password SCIM Bridge on Azure Container Apps
+
+This deployment example describes how to deploy 1Password SCIM Bridge as an app on Azure's [Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service.
+
+The deployment consists of two [containers](https://learn.microsoft.com/en-us/azure/container-apps/containers), the SCIM bridge container and a Redis container and an [ingress](https://learn.microsoft.com/en-us/azure/container-apps/ingress-overview) for the SCIM bridge container.
+
+## In this folder
+
+- [`README.md`](./README.md): the document that you are reading.
+
+## Overview
+
+Deploying 1Password SCIM Bridge on Azure Container Apps comes with a few benefits:
+
+- For standard deployments, Azure Container Apps will host your SCIM bridge for ~$16 USD/month (when this document was written)
+  > **Note**
+  >
+  > Conatiner Apps pricing is variable based on activity. Details can be found on [Microsoft's pricing page](https://azure.microsoft.com/en-us/pricing/details/container-apps/).
+- No DNS management, and an obsecure DNS record. Azure Container Apps automatically provides a unique URL for your SCIM bridge.
+- Azure Container Apps automatically handles TLS certificate management on your behalf.
+- You will deploy 1Password SCIM Bridge directly to Azure from your Azure Portal or some commands can be performed from your local terminal. There is no requirement to clone this repository for this deployment.
+
+## Prerequisites
+
+- A 1Password account with an active 1Password Business subscription or trial
+  > **Note**
+  >
+  > Try 1Password Business free for 14 days: <https://start.1password.com/sign-up/business>
+- An Azure account with permissions to create a Container App.
+  > **Note**
+  >
+  > If you don't have a Azure account, you can sign up for a free trial with starting credit: <https://azure.microsoft.com/en-us/free/>
+- [Azure CLI tool](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) with the ContainerApp Extension or access to the [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/quickstart) if performing the command driven deployment.
+
+## Getting started
+
+### Step 1: Generate credentials for automated user provisioning with 1Password
+
+1. [Sign in](https://start.1password.com) to your account on 1Password.com.
+2. Click [Integrations](https://start.1password.com/integrations/directory) in the sidebar.
+3. Choose your identity provider from the User Provisioning section.
+4. Choose "Custom deployment".
+5. Use the "Save in 1Password" buttons for both the `scimsession` file and `bearer token` to save them as items in your 1Password account. 
+6. Also use the download icon next to the `scimsession` file to save this file on your system.
```

Comment:
Drop `also`. It's a step, so it's clear that it needs to be done

Reply:
Applying @plttn suggestion.

```suggestion
6. Use the download icon next to the `scimsession` file to save this file on your system.
```